### PR TITLE
Login: rework ui

### PR DESCRIPTION
**Pull request by tkohr**
PR intends to refine the login UI based on the work done in MEL project.

It includes only styling part, no error handling or translation capabilities.

Might need rebase on https://github.com/georchestra/georchestra-gateway/pull/27. 

I see that there are two static folders on different levels, not sure how they are used. Also is the `index.html` essential?

### CODE REVIEW

File: gateway/src/main/resources/templates/login.html [hunk 1]
```diff
@@ -5,18 +5,24 @@
...
-    <title>Login</title>
-    <link href="https://maxcdn.bootstrapcdn.com/bootstrap/4.0.0-beta/css/bootstrap.min.css" rel="stylesheet" integrity="sha384-/Y6pD6FV/Vv2HJnA6t+vslU6fwYXjCFtcEpHbNJ0lyAFsXTsjBbfaDjzALeQsN6M" crossorigin="anonymous">
-    <link href="https://getbootstrap.com/docs/4.0/examples/signin/signin.css" rel="stylesheet" crossorigin="anonymous"/>
+    <title>Please sign in</title>
+    <script src="https://cdn.jsdelivr.net/gh/georchestra/header@dist/header.js"></script>
+    <link href="login/bootstrap.min.css" rel="stylesheet">
+    <link href="login/signin.css" rel="stylesheet"/>
 </head>
 <body>
+<!--
```
[the remark this answers is not in the document]
Comment (pmauduit) [reply gateway/src/main/resources/templates/login.html:14]: > Does this seem sufficient for the existing use cases @pmauduit ?

yes, I'd expect

**Review by pmauduit — APPROVED** [review]

**Comment by jeanmi151** [comment]
@tkohr @emmdurin  can we merge ?

**Comment by tkohr** [comment]
To me, it looks good. @emmdurin anything to add?

**Comment by emmdurin** [comment]
OK for me.

**Pull request merged**